src/datumaro/components/transformer.py

```diff
@@ -2,13 +2,20 @@
 #
 # SPDX-License-Identifier: MIT
 from multiprocessing.pool import ThreadPool
-from typing import Iterator, List, Optional
+from typing import Dict, Iterator, List, Optional, Type
 
 import numpy as np
 
+from datumaro import MediaElement
```

Comment:
```suggestion
from datumaro.components.media import MediaElement
```
`datumaro` package is not initialized at this point for imports

Reply:
fixed